sysboundary/outflow.cpp

```diff
@@ -359,54 +359,40 @@ namespace SBC {
       determineFace(&isThisCellOnAFace[0], x, y, z, dx, dy, dz, true);
       
       cuint sysBoundaryLayer = technicalGrid.get(i,j,k)->sysBoundaryLayer;
+      if(sysBoundaryLayer == 1) {
+         cint neigh_i=i + ((component==0)?-1:0);
+         cint neigh_j=j + ((component==1)?-1:0);
+         cint neigh_k=k + ((component==2)?-1:0);
+         cuint neighborSysBoundaryFlag = technicalGrid.get(neigh_i, neigh_j, neigh_k)->sysBoundaryFlag;
       
```

Comment:
Do we really want to ignore these flags?
this->facesToProcess[i]
!this->facesToSkipFields[i]

Reply:
The first I guess yes, the other should be integrated in the else part I think.